docs/dev/reference/apis/services/vision.md

```diff
@@ -81,6 +85,10 @@ See our guide [Navigate with a Rover Base](/tutorials/services/navigate-with-rov
 
 Any camera that can return 3D pointclouds can use 3D object segmentation.
 
+{{% alert title="Tip" color="tip" %}}
+3D segmentation operations often require [frame system](/operate/reference/services/frame-system/) configuration to properly relate camera coordinates to your machine's spatial reference frames. This enables the vision service to provide meaningful 3D coordinates and spatial relationships.
```

Comment:
```suggestion
3D segmentation operations require [frame system](/operate/reference/services/frame-system/) configuration to properly relate camera coordinates to your machine's spatial reference frames.
This enables the vision service to provide meaningful 3D coordinates and spatial relationships.
```
[nit] newline
Less sure about the "often" being necessary--I defer to your expertise Nathan unless SME is necessary